lms/static/scripts/frontend_apps/utils/onedrive-picker-client.js

```diff
@@ -32,9 +32,10 @@ export class OneDrivePickerClient {
     const oneDrive = await this._oneDriveAPI;
     return new Promise((resolve, reject) => {
       const success = (/** @type {any} */ file) => {
+        const name = file.value[0].name;
```

Comment:
Alternatively, but I don't know if that's more or less obvious.
```suggestion
        const {name, permissions} = file.value[0];
```